auth/auth.go

```diff
@@ -80,7 +79,7 @@ type (
 )
 
 // @TODO (jay-dee7) maybe a better way to do it?
-func (a *auth) StateTokenCleanup() {
+func (a *auth) stateTokenCleanup() {
 	// tick every 10 minutes, delete ant oauth state tokens which are older than 10 mins
```

Comment:
you mean seconds?

Reply:
Yes it should be seconds since it ticks every 10 seconds. But the the timer within it expires after 10 mins. So it checks every 10 seconds for any stale tokens ie. delete every 10 seconds, tokens that are older than 10 mins